src/core/lib/gprpp/inlined_vector.h

```diff
@@ -100,7 +89,27 @@ class InlinedVector {
 
   size_t size() const { return size_; }
 
+  void clear() {
+    destroy_elements();
+    dynamic_ = nullptr;
+    size_ = 0;
+    dynamic_capacity_ = 0;
```

Comment:
with this change, that'd be two places where initialization of member variables happen. If new ones are added to the class, we'd have to remember adding them here. I suggest creating a private aux method to initialize the three member variables, to be invoked here and in the constructor.

Reply:
Done.